lib/index.coffee

```diff
@@ -30,9 +41,10 @@ lint = (editor) ->
     throw new Error stderr or stdout unless typeof parsed is 'object'
     (parsed.files?[0]?.offenses or []).map (offense) ->
       {cop_name, location, message, severity} = offense
+      {cop_name, message} = extractUri cop_name, message
```

Comment:
`message` isn't being modified in this function, there's no need to return it here.